### PR TITLE
Fix example format in Ansible modules

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#examples-block
```
Per playbook best practices, each example should include a name: line:
EXAMPLES = r'''
- name: Ensure foo is installed
  modulename:
    name: foo
    state: present
'''
The name: line should be capitalized and not include a trailing dot.
```

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

**Comment by Andersson007** [comment]
ready_for_review

**Comment by Andersson007** [comment]
CC @samccann @acozine

File: lib/ansible/modules/fail.py [hunk 1]
```diff
@@ -38,8 +38,8 @@
 '''
 
 EXAMPLES = r'''
-# Example playbook using fail and when together
-- fail:
+- name: Example playbook using fail and when together
```
Comment (felixfontein) [inline lib/ansible/modules/fail.py]: This is a task, not a playbook :)
```suggestion
- name: Example using fail and when together
```

File: lib/ansible/modules/pip.py [hunk 2]
```diff
@@ -134,96 +134,98 @@
...
-# Install specified python requirements offline from a local directory with downloaded packages.
-- pip:
+- name: Install specified python requirements offline from a local directory with downloaded packages
+  pip:
     requirements: /my_app/requirements.txt
     extra_args: "--no-index --find-links=file:///my_downloaded_packages_dir"
 
-# Install (Bottle) for Python 3.3 specifically,using the 'pip3.3' executable.
-- pip:
+- name: Install bottle for Python 3.3 specifically,using the 'pip3.3' executable
```
Comment (felixfontein) [inline lib/ansible/modules/pip.py]:
```suggestion
- name: Install bottle for Python 3.3 specifically, using the 'pip3.3' executable
```

File: lib/ansible/modules/pip.py [hunk 3]
```diff
@@ -134,96 +134,98 @@
...
+- name: Install multi python packages with version specifiers
+  pip:
     name:
       - django>1.11.0,<1.12.0
       - bottle>0.10,<0.20,!=0.11
 
-# Install python package using a proxy - it doesn't use the standard environment variables, please use the CAPITALIZED ones below
-- pip:
+# It doesn't use the standard environment variables, please use the CAPITALIZED ones below
+- name: Install python package using a proxy
```
Comment (felixfontein) [inline lib/ansible/modules/pip.py]:
```suggestion
- name: Install python package using a proxy
  # Pip doesn't use the standard environment variables, please use the CAPITALIZED ones below
```

**Comment by Andersson007** [comment]
@felixfontein done, thanks for reviewing!

**Review by felixfontein — APPROVED** [review]
shipit

**Comment by acozine** [comment]
Thanks @Andersson007 these examples look much better.

**Pull request merged**